inc/manager.php

```diff
@@ -481,7 +481,7 @@ public function process_urls_from_content( $html ) {
 	 * @return array
 	 */
 	public function extract_image_urls_from_content( $content ) {
-		$regex = '/(?:http(?:s?):)(?:[\/\\\\|.|\w|\s|-])*\.(?:' . implode( '|', array_keys( Optml_Config::$extensions ) ) . ')(?:\?{1}[\w|=|&|\-|\.|:|;]*)?/';
+		$regex = '/((?:http(?:s?):)(?:[\/\\\\|.|\w|\s|-])*\.(?:' . implode( '|', array_keys( Optml_Config::$extensions ) ) . ')(?:\?{1}[\w|=|&|\-|\.|:|;]*)?)|((?:[\/\\\\|.|\w|\s|-])*\.(?:' . implode( '|', array_keys( Optml_Config::$extensions ) ) . ')(?:\?{1}[\w|=|&|\-|\.|:|;]*)?)/';
```

Comment:
this seems not right if we try to do target in regex both with and without schema urls, why we don't use just the regex without the schema to search. 

Also, how the re-written urls will look, they will include the schema of the original url?

Reply:
ok, i will test that 

yes, they will include the schema

Reply:
@GrigoreMihai You can check the regex here https://regex101.com/ and add the link with the tests to the conversation.

Reply:
https://regex101.com/r/LNLqzC/1